.travis.yml

```diff
@@ -22,6 +22,7 @@ addons:
     - svn-14671.prod.hosting.acquia.com
     - svn-29892.prod.hosting.acquia.com
     - svn-37631.prod.hosting.acquia.com
+    - svn-37631.prod.hosting.acquia.com
```

Comment:
need to make this unique

Reply:
I guess since its the same as 02, it can just be left out for now.

Reply:
cool! I'll update it accordingly! Thank you!